test/query_component_test.dart

```diff
@@ -249,18 +254,32 @@ void main() {
         ];
       });
 
-      test(
-          'highlight underpacing unchecked invokes populate(multiple-IO, false)',
+      tearDown(() => clearInteractions(mockQueryService));
```

Comment:
this shouldnt be necessary if you are making a new instance for every test

Reply:
I'm using singleton pattern (needs component provider overwrite) here so it always returns the same instance.